src/main/java/seedu/address/logic/commands/ClientNoteEditCommand.java

```diff
@@ -66,10 +66,10 @@ public CommandResult execute(Model model) throws CommandException {
         accumulatedTags.addAll(newNote.getTags());
         // because parser used tagNoteMap#getUniqueTags, it is okay for there to be duplicates in previous tags and
         // new Note's tags. Overwriting will keep one of the two duplicates, and they are the same object reference.
-        newNote.setTags(accumulatedTags);
-        if (accumulatedTags.size() <= 1 && accumulatedTags.contains(Tag.UNTAGGED)) {
+        if (accumulatedTags.size() > 1 && accumulatedTags.contains(Tag.UNTAGGED)) {
```

Comment:
no need for contains, since set.remove won't throw any errors/exception even if the tag is not inside